config/environments/development.rb

```diff
@@ -35,7 +35,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :cloudinary
```

Comment:
for our seed I want to use local directory to save the result PDF, I don't know how to use the active_storage as local and cloudinary at the same time.

Reply:
According to Trouni, pdf can be stored with Cloudinary.